src/diffusers/models/unet_2d_condition.py

```diff
@@ -63,6 +63,62 @@ class UNet2DConditionOutput(BaseOutput):
     sample: torch.FloatTensor = None
 
 
+class FourierEmbedder(nn.Module):
```

Comment:
Can we move this to https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/embeddings.py